config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-release-v1beta1.yaml

```diff
@@ -11,7 +11,7 @@ periodics:
   extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.4
+      base_ref: release-1.5
```

Comment:
Do you plan to update references to release-1.4 in cloud-provider-azure jobs in a separate PR?

Reply:
Yeah

Reply:
lol I forgot I already did this:

https://github.com/kubernetes/test-infra/pull/27492